src/main/java/com/dev/cinema/controller/CinemaHallController.java

```diff
@@ -0,0 +1,32 @@
+package com.dev.cinema.controller;
+
+import com.dev.cinema.model.CinemaHall;
+import com.dev.cinema.model.dto.CinemaHallRequestDto;
+import com.dev.cinema.model.dto.CinemaHallResponseDto;
+import com.dev.cinema.model.mapper.CinemaHallMapper;
+import java.util.List;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/cinema-halls")
+public class CinemaHallController {
+    private final CinemaHallMapper cinemaHallMapper;
+
+    public CinemaHallController(CinemaHallMapper cinemaHallMapper) {
+        this.cinemaHallMapper = cinemaHallMapper;
+    }
+
+    @PostMapping(value = "/add")
```

Comment:
```suggestion
    @PostMapping
```
![image](https://user-images.githubusercontent.com/50640794/84403003-dc5f3200-ac0d-11ea-9aea-c1d5b6bbe7b2.png)